lab-justin/lib/storage.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
It doesn't look like you're using this module anywhere--all the require statements for it have been commented out.